packages/deployer/src/compiler.ts

```diff
@@ -238,8 +288,8 @@ export class Compiler {
         if (_.isUndefined(this._contractSourcesIfExists)) {
             throw new Error('Contract sources not yet initialized');
         }
-        const contractBaseName = path.basename(importPath);
-        const source = this._contractSourcesIfExists[contractBaseName];
+        const fileName = path.basename(importPath);
+        const source = this._contractSourcesIfExists[fileName];
```

Comment:
Rename `source` to `sourceIfExists`

Reply:
Hmm, `source` should always exist here assuming that `this._contractSourcesIfExists` is initialized.

Reply:
Yep, it's checked above, my bad!